pkg/cstor/poolcluster/v1alpha1/raidgroups/build.go

```diff
@@ -60,6 +60,22 @@ func (b *Builder) WithReadCache() *Builder {
 	return b
 }
 
+// WithCSPCBlockDeviceList sets the blockdevices field with provided value
+func (b *Builder) WithCSPCBlockDeviceList(cspcBDList []*apisv1alpha1.CStorPoolClusterBlockDevice) *Builder {
+	if cspcBDList == nil {
```

Comment:
can we have `len(cspcBDList) == 0` instead as the slice will never be nil

Reply:
done